src/util/email.ts

```diff
@@ -0,0 +1,73 @@
+// Load Environment Variables
+import dotenv from 'dotenv';
+dotenv.config();
+
+import Email from 'email-templates';
+import { createTransport } from 'nodemailer';
+import SMTPTransport from 'nodemailer/lib/smtp-transport';
+import path from 'path';
+
+const getTransportOptions = (): SMTPTransport.Options | undefined => {
+  //With SMTP host/port we assume local development mailing
+  if (process.env.EMAIL_CONFIG_USERNAME && process.env.EMAIL_CONFIG_PASSWORD) {
+    return {
+      name: 'CubeCobra.com',
+      secure: true,
+      service: 'Gmail',
+      auth: {
+        user: process.env.EMAIL_CONFIG_USERNAME,
+        pass: process.env.EMAIL_CONFIG_PASSWORD,
+      },
+    };
+  } else if (process.env.EMAIL_CONFIG_SMTP_HOST && process.env.EMAIL_CONFIG_SMTP_PORT) {
```

Comment:
The goal is to use a local mail server here such as mailpit with docker. That is an easier setup than getting developers to create app passwords for their gmail (in large part because the google UI for it sucks)